packages/twenty-server/src/modules/workflow/common/standard-objects/workflow-version.workspace-entity.ts

```diff
@@ -98,7 +98,7 @@ export class WorkflowVersionWorkspaceEntity extends BaseWorkspaceEntity {
     icon: 'IconSettingsAutomation',
   })
   @WorkspaceIsNullable()
-  steps: WorkflowStep[] | null;
+  steps: WorkflowAction[] | null;
```

Comment:
do you think it worth renaming `steps` into `actions`?

Reply:
I don't think branches, conditions, loops will be actions. There would be better as steps. Wdyt?

Reply:
indeed